code/datums/mutations/speech.dm

```diff
@@ -73,11 +73,9 @@
 		message = " [message] "
 		//Time for a friendly game of SS13
 		message = replacetext(message," stupid "," smart ")
-		message = replacetext(message," retard "," genius ")
 		message = replacetext(message," unrobust "," robust ")
 		message = replacetext(message," dumb "," smart ")
 		message = replacetext(message," awful "," great ")
-		message = replacetext(message," gay ",pick(" nice "," ok "," alright "))
```

Comment:
Keep this in

Reply:
I rather not, one it isn't used, and this mutation says the opposite of what it means, meaning it uses the word in a negative light, which is pretty cringe.